package.json

```diff
@@ -35,8 +35,7 @@
     "validate-commits": "./bin/devkit-admin validate-commits",
     "prepush": "node ./bin/devkit-admin hooks/pre-push",
     "preinstall": "node ./tools/yarn/check-yarn.js",
-    "webdriver-update-appveyor": "webdriver-manager update --standalone false --gecko false --versions.chrome 2.37",
-    "webdriver-update-circleci": "webdriver-manager update --standalone false --gecko false --versions.chrome $CHROMEDRIVER_VERSION_ARG "
+    "webdriver-update": "webdriver-manager update --standalone false --gecko false --versions.chrome 2.45"
```

Comment:
Hard coding the version in multiple places should be avoided if possible.  Would it be possible to centralize the chrome and driver version information?

Reply:
Hmm no ergonomic solution comes to mind. Storing it on a env var would need the var to be set, and a cross platform solution.

Storing it on a node module would require a roundabout way of calling `webdriver-update` from the node bin.

Right now it's in 2 places, which is better than before, but still would be better to be in a single place. Although to be fair it's on two places because each of them happen to install a certain version of puppeteer, and these two things happen to match (e.g. one can imagine one of these places using a different puppeteer/webdriver combo).

Did you have an approach in mind?